common/app/common/editions/International.scala

```diff
@@ -0,0 +1,143 @@
+package common.editions
+
+import java.util.Locale
+
+import common._
+import conf.Switches
+import conf.Switches.InternationalEditionBetaSwitch
+import org.joda.time.DateTimeZone
+
+object International extends Edition(
+  id = "INT",
+  displayName = "International",
+  timezone = DateTimeZone.forID("Europe/London"),
+  locale = Locale.forLanguageTag("en"),
+  homePagePath = "/international",
+  editionalisedSections = Seq("") // only the home page
+){
+
+  implicit val INT = International
+
+  override def isBeta: Boolean = InternationalEditionBetaSwitch.isSwitchedOn
+
+  override val cookieValue: String = "INTL"
```

Comment:
do we really need to separate out the cookieValue, ID and the path?  It does add complexity so it would be nice if we can postpone that addition, but maybe we're already past the point of no return?